src/post.ts

```diff
@@ -0,0 +1,10 @@
+import {EventPayload} from './EventPayload';
+
+const post = (
```

Comment:
nit: can we add a comment explaining how this will be used? Also, maybe consider marking it as a to-do stating that this will change significantly.

Reply:
Done. I also included in the todo that we should also update the description.